#include "py/runtime.h"
#include <pbio/error.h>

void pb_assert(pbio_error_t error);
